tests/SIL.LCModel.Tests/DomainServices/WritingSystemServicesTests.cs

```diff
@@ -291,6 +291,7 @@ public void UpdateWritingSystemListField_RemovesWsCode()
 			testEntry.ReversalIndex.WritingSystem = "fr";
 			testEntry.ReversalForm.set_String(m_wsFr, "fr");
 			WritingSystemServices.UpdateWritingSystemFields(Cache, "fr", "blz");
+			WritingSystemServices.UpdateWritingSystemFields(Cache, "fr", null);
```

Comment:
I think we should modify this to be Assert.DoesNotThrow(()=> WritingSystemServices...);

So that we know what this line in the test is proving.

Reply:
Yes Jason, I added Assert for my change.